src/App.vue

```diff
@@ -1,12 +1,41 @@
 <template>
   <div id="app">
-    <h1>hello world</h1>
+    <tx-preloader v-if="preload" :progress="progress" />
+
+    <main v-else>
+      <h1>hello world</h1>
+    </main>
   </div>
 </template>
 
 <script>
-export default {}
+import Preloader from "./assets/components/atoms/Preloader.vue";
+
+export default {
+  components: {
+    "tx-preloader": Preloader,
+  },
+  data() {
+    return {
+      preload: true,
+      progress: 0,
+    };
+  },
+  mounted() {
+    const preloadInterval = setInterval(() => {
+      if (this.progress < 100) {
+        this.progress++;
+      } else {
+        clearInterval(preloadInterval);
+        this.preload = false;
+      }
+    }, 10);
+  },
+};
 </script>
 
 <style lang="scss" scoped>
+main {
```

Comment:
please use class. Reference: https://v2.vuejs.org/v2/style-guide/?redirect=true#Element-selectors-with-scoped-use-with-caution

Reply:
thank you for the advice, soon will be fixed